cernopendata/modules/theme/assets/semantic-ui/js/search/components/RangeAggregation.js

```diff
@@ -0,0 +1,206 @@
+import _debounce from 'lodash/debounce';
+import PropTypes from "prop-types";
+import React, { useState, useMemo, useEffect, useRef, useCallback } from "react";
+import { useSelector } from "react-redux";
+import { FlexibleWidthXYPlot, VerticalRectSeries, Hint } from "react-vis";
+import { Card } from "semantic-ui-react";
+import RangeSlider from './RangeSlider';
+import { withState } from 'react-searchkit';
+import { safeParse } from "../utils";
+
+const HALF_BAR_WIDTH = 0.4;
+const SELECTED_COLOR = "#91d5ff";
+const DESELECTED_COLOR = "#bfbfbf";
```

Comment:
Why are some colors defined here, and some in the css? Would it be possible to have all of them in the scss?